jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/container/TabSheetLoader.java

```diff
@@ -53,11 +61,43 @@ public void loadComponent() {
     protected void createTabs(Element element) {
         LayoutLoader loader = getLayoutLoader();
 
+        boolean firstTab = true;
         for (Element subElement : element.elements("tab")) {
-            ComponentLoader<?> componentLoader = loader.getLoader(subElement, TabLoader.class);
-            componentLoader.initComponent();
+            boolean shouldBeLazy = loadBoolean(subElement, "lazy").orElse(false) && !firstTab;
+            ComponentLoader<?> componentLoader = loader.getLoader(subElement,
```

Comment:
Here we already know if a tab is lazy, so we can skip creating `Loader` and putting it in `pendingLoadComponents`. If a tab is `lazy`, it'd be more efficient to store an `element` rather than a `loader` instance. A loader can be created in the `selectedTabChangeHandler` on demand.